lib/cdo/user_agent_parser.rb

```diff
@@ -2,7 +2,7 @@
 
 class UserAgentParser::UserAgent
   def mobile?
-    name =~ /Mobile/
+    !(name =~ /Mobile/).nil?
```

Comment:
CloudFront doesn't pass the User Agent through -- we'll need to do this client side.  Opened Pivotal #108503362 to track.